index.js

```diff
@@ -413,14 +413,16 @@ alexa.session = function(session) {
 
 alexa.router = function(app, request, response, request_json) {
   this.intent = function(intent) {
+    request.intent = intent;
```

Comment:
This modifies the incoming request, I don't think that's a good idea, at the very least it's an anti-pattern to be modifying something coming from the outside.

Reply:
Hmm, i understand and already updated the code as that field is still accessible from the raw data